lib/src/model/modal_config.dart

```diff
@@ -21,6 +21,9 @@ class S2ModalConfig {
   /// Whether the option list is filterable or not
   final bool useFilter;
 
+  /// Whether the filter is autocomplete or need confirmation
+  final bool autofilter;
```

Comment:
I guess its better to use `filterAutocomplete` or `filterAuto` or `filterOnChanged` to keep consistency with `filterHint`

Reply:
Fixed, is `filterAuto` now.